scripts/get-version

```diff
@@ -47,11 +46,23 @@ else
     PATCH=0
 fi
 
+# if we're in a features/xxx branch and caller passed --embed-feature-branch then append `-xxx` to
+# the version as well.
+FEATURE_TAG=""
+for arg in "$@"
+do
+    if [[ "$arg" == "--embed-feature-branch"]]; then
+        if [[ "${TRAVIS_BRANCH:-}" == features/* ]]; then
+            FEATURE_TAG=$(echo "${TRAVIS_BRANCH}" | sed -e 's|^features/|-|g')
+        fi
+    fi
+done
```

Comment:
changed to only add the feature-branch tag if requested by caller.  only dotnet requests this.